etc/configuration/default.json

```diff
@@ -152,48 +152,30 @@
   },
   "robot_detection": {
     "vision_top": {
+      "enable": false,
```

Comment:
Why do you disable the robot detection here? For testing the foot detection?

Reply:
The robot detection is currently not integrated in the filter. This will be done in a future PR. At the moment, the robot detection is disabled, as it is not used and otherwise would just burn execution time ;)